src/resources/PackageResource.ts

```diff
@@ -0,0 +1,123 @@
+import { Response } from 'node-fetch';
+
+import { ExportedAuditTrail, Package, Requests, Responses } from '../types';
+import { serializeDate } from '../utils/serializeDate';
+import { Resource } from './Resource';
+
+/**
+ * Resource class to interact with
+ * {@link https://community.onespan.com/documentation/onespan-sign/guides/sender-guides/user/transactions | packages (transactions)}.
+ *
+ * @public
+ */
+export class PackageResource extends Resource {
+  /**
+   * Creates a package (transaction).
+   *
+   * @param payload - Package initial information
+   * @returns A payload that contains the ID of the newly created package.
+   *
+   * @remarks
+   * - {@link https://community.onespan.com/products/onespan-sign/sandbox#/Packages/api.packages.post | REST API documentation (OneSpan)}
+   *
+   * - {@link https://community.onespan.com/documentation/onespan-sign/guides/feature-guides/developer/creating-transaction-sender | Creating a Transaction for a Sender (OneSpan)}
+   */
+  public async create(payload: Requests.PackageData): Promise<Responses.CreatePackage> {
```

Comment:
Would it be more clear to have `create(payload: Requests.CreatePackage)`? That way it would match with `Responses.CreatePackage`.

Reply:
`PackageData` is also used in `update` so it won't make much sense for it to use `Requests.CreatePackage`. However, I can probably use type aliases for them

Reply:
I'd add a type alias for clarity and in the case they diverge for some reason in the future.

Reply:
Done ed8b5d1